util/data_utils.py

```diff
@@ -10,6 +10,10 @@ def __init__(self, filename, transform=None, transform_y=None, device='cpu'):
     loaded_file = torch.load(filename, map_location=torch.device(device))
     self.data = loaded_file[0]
     self.condition = torch.as_tensor(loaded_file[1]).float()
+    if len(loaded_file) > 2:
```

Comment:
I take it this is some sort of bucket weighting? Seems fine as you've written it as a supplementary input file

Reply:
It's according to the energy weighting, that has been tested months ago. No improvement observed, but may be good to keep it as it may be needed in the future.